### ISSUE TITLE
symbols and responsiveness on selecting a process

### ISSUE THREAD

**Issue by didito** [VerySleepy/verysleepy#44]
I set up symbols and it works fine in general.

But selecting a different process in the "Select a process to profile" suddenly became a rather lengthy procedure. From the initial wait on all the DBGHELP / SYMSRV shenanigans (5 - 10s, depending on latency to symbol server I guess) to a "Not Responding" window to an eventual hang or refresh it really gets in my way (20s in total) when I am more focused on profiling the startup of a certain process.

I don't want to give up symbolication yet and there is obviously ways around this behavior (suspending process in Process Explorer or have a wait loop in your code) but I wonder if this symbol loading for the translation can't be done in a more lazy / deferred way?
I really would love the list to be more responsive, i.e. selecting a newly spawned process has no delay and just cancels what it is currently doing ...

**Comment by tim-lebedkov** [comment]
A similar issue here while selecting my program Npackd with a .pdb file. CPU usage goes to 100%. See the attached image for a typical stack trace.
![stacktrace](https://user-images.githubusercontent.com/831371/28018780-dc107bd4-657e-11e7-953d-fb9ecce39616.png)

**Comment by AlanIWBFT** [comment]
If you're on windows its probably due to the bundled dbghelpms.dll is faulty and freezes indefinitely, I noticed that copying a new dbghelp.dll from your Windows SDK and rename it to debughelpms.dll can help a lot.

**Comment by CyberShadow** [comment]
Could you please see if this issue occurs with the latest release:
https://github.com/VerySleepy/verysleepy/releases

#65 was merged, which I understand tried to address this issue. If you encounter further problems, please leave a comment or open a new issue. Thanks!
